dimod/core/initialized.py

```diff
@@ -19,26 +19,31 @@
 this abstract base class in a sampler.
 
 """
-
 import abc
 
 from collections import namedtuple
 from numbers import Integral
 
+import typing
+
 import numpy as np
 
 from dimod.sampleset import as_samples, infer_vartype, SampleSet
 from dimod.vartypes import Vartype
+from dimod.binary.binary_quadratic_model import BinaryQuadraticModel
 
-__all__ = ['Initialized']
-
+try:
+    InitialStateGenerators = typing.Literal['none','tile', 'random']
```

Comment:
Missing a space. Also probably should be `InitialStateGenerator` (I know that I had it plural in my example).